Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -304,19 +305,29 @@ private sealed class CacheContext : IDisposable
             public readonly CacheProcessor Sut;
             public readonly AnalysisCacheMsg Cache = new();
 
+            // The paths in the cache are:
+            // - serialized with Unix separator `/`
+            // - are part of a directory structure
+            // The tests should reflect that.
+            private readonly FileData[] fileData =
```

Comment:
This is only accessed inside CacheContext constructor. Can be a local variable there instead.